server/sonar-server/src/test/resources/org/sonar/server/component/ws/SearchViewComponentsActionTest/shared.xml

```diff
@@ -4,11 +4,13 @@
   <group_roles id="1"
                group_id="[null]"
                resource_id="100"
-               role="user"/>
+               role="user"
+               organization_uuid="[null]"/>
```

Comment:
this commit adds the column, so I prefer having all db tests aware of it, even if value is always null.